book/first-figure.ipynb

```diff
@@ -0,0 +1,512 @@
+    "* Run cell: `Shift`+`Enter`\n",
+    "* Code indentation: `TAB`\n",
+    "* Code dedenting: `Shift`+`TAB`\n",
+    "* Writing single-line comments: begining with **#** or `Command`+`/`\n",
+    "* Auto-completion: `TAB` while you writing some code\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7c1f7df7-5db9-4301-8dd2-05ce19fb3b28",
+   "metadata": {
+    "panel-layout": {
+     "height": 119.774,
+     "visible": true,
+     "width": 100
+    }
+   },
+   "source": [
+    "## Importing \n",
+    "\n",
+    "First thing to do is load PyGMT (`import`) so that we can access its functionality. \n",
+    "\n",
+    "PyGMT has a flat package layout, meaning that you can access everything in it with a single `import`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "71301b84-c2d0-4752-87f6-bed981a19313",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pygmt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ce622686-4279-4ab3-b00a-a9f3e7dc8cd4",
+   "metadata": {},
+   "source": [
+    "## Starting your first figure -- [`pygmt.Figure`](https://www.pygmt.org/latest/api/generated/pygmt.Figure.html#pygmt.Figure)\n",
+    "Every figure must start with the creation of a pygmt.Figure"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ca870b9d-4b21-420a-b547-4983a208d3eb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = pygmt.Figure()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3336150b-5a93-4f03-b355-9a56705581ab",
+   "metadata": {},
+   "source": [
+    "We have blank canvas in the `fig` variable now, we will overlay elements and show it on same `fig`. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03827f08-f6f5-4056-b73b-2071c90d8f0e",
+   "metadata": {},
+   "source": [
+    "## Drawing coastline -- [`figure.coast`](https://www.pygmt.org/dev/api/generated/pygmt.Figure.coast.html)\n",
+    "Before plotting, you need to define \n",
+    "1. `region`: controling the geographic or Cartesian extent of the figure. Regions are specified as lists of `[xmin, xmax, ymin, ymax]`. Another shortcut, `region='g'`, which specifies a **global** domain.\n",
+    "2. `projection`: controling the map projection. GMT (and therefore PyGMT) supports 31 different map projections, from basic Cartesian axes to [arbitrary vertical perspectives](https://docs.generic-mapping-tools.org/latest/gallery/ex26.html). Again, if you're familiar with GMT, these are specified using the `-J` flag, as in `-JM4i`. In PyGMT, projections codes are still strings, but the `J` is dropped, so an equivalent PyGMT argument would be `projection='M4i'`.\n",
+    "\n",
+    "> Determining the size of your images `4i`, the size will affect how large of your following fontsize and markersize\n",
+    ">> **length unit** `c`: cm / `i`: inch(2.54 cm) / `p`: point(1/72 of an inch)\n",
```

Comment:
> My only suggestion is to add a note saying something like: "If you use -JM4i, then the width of your map will be 4 inches. The real width of the figure could be bigger if you add elements (like a frame, colorbar, etc.) outside the map."

If I may add to @Esteban82's comment, it would be good to use SI units (centimeters) by default (PyGMT has this convention since https://github.com/GenericMappingTools/pygmt/pull/795). Still ok to mention **i**nches, and **p**oint, but the -J flag should use something like `-JM2c`.

Reply:
I think we still can mention **c**, **i**, and **p** since users will use **p** for font text.